docs/content/migration.md

```diff
@@ -0,0 +1,535 @@
+- Both APIs include [CSS Grid][65] features in addition to the CSS Flexbox features from
+  AngularJS Material.
+- The ability to define [Custom Breakpoints][118]
+- A [MediaObserver Service][119] that allows subscribing to `MediaQuery` activation changes
+
+The [Angular Layout][120] library's full UMD bundle is around `275` KBs. This size can be reduced by
+consuming the various modules and using the Angular CLI to remove unused library code, but this
+may be more complexity than you need.
+
+#### CDK Layout
+
+The Angular CDK's [Layout][121] module provides:
+- A set of `Breakpoints` that match the [Material Design][122] responsive layout grid breakpoints
+- A `BreakpointObserver` Service that provides
+    - the ability to check media query matching state via `isMatched()`
+    - the ability to subscribe to a stream of events based on changes to an array of media queries
+      that are passed into `observe()`
+- A `MediaMatcher` Service that provides access to the low level, native [`MediaQueryList`][123]
+
+If you are building responsive components or need responsive features in your application with a
+minimal increase in bundle size, then the CDK's Layout module is an option you should consider.
+
+### Typography
+
+AngularJS Material [typography classes][107] are generally identical to the Angular Material
+[typography classes][27].
+
+Angular Material also adds the following typography features:
+- [Customization via Sass mixins][174] that are similar to the theme configuration mixins
+- Utility Sass mixins and functions for use in [styling your custom components][175]
+  
+## Comparison of Features and Migration Tips
+
+Angular Material introduces a number of [new components](#new-components) and features that do not
+exist in AngularJS Material. Angular Material also makes greater use of semantic HTML and native
+elements to improve accessibility.
+
+However, some directives and services have not been implemented in Angular Material. These are
+indicated by an `N/A` in the New Docs column of the feature tables found below. 
+
+The Notes column, in each row of the tables, includes information like the new name of the feature
+or an alternative approach to implementing the feature. If the notes are blank, then the name of
+the old feature and the new feature are nearly identical (the APIs are likely to be different).
+
+### Directives
+Most directives changed to components. These make for straight forward conversions, for instance
+`<md-card>` changed to `<mat-card>`. Some element directives changed to attribute directives, for
+instance `<md-subheader>` changed to `matSubheader`.
+
+Here is a reference table where you can review and compare the AngularJS Material and
+Angular Material documentation. AngularJS Material uses the `md-` and `$md` prefixes while Angular
+Material uses the `mat-`, `mat`, and `Mat` prefixes. The `md-` prefixes have been omitted to make
+this table more readable.
```

Comment:
I would replace this paragraph with just the header `#### Component comparison reference` and then just tweak the column headers to be ``Old (`md-`)`` and ``New (`mat-`)``

Reply:
I'll tweak this a bit, but the suggested approach seems to omit (or leave the reader to assume) quite a bit of information.